.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on: push
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version:
+          - '2.4'
```

Comment:
Remove 2.4? Especially since you're saying bellow we're testing on 2.5+.

Reply:
I’ll do it in a separate pull request.